core/templates/dev/head/pages/exploration_editor/translation_tab/TranslationTabActiveContentIdService.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to get and set active content id in translation tab.
+ */
+
+oppia.factory('TranslationTabActiveContentIdService', [
+  '$log', '$rootScope', 'StateContentIdsToAudioTranslationsService',
+  function($log, $rootScope, StateContentIdsToAudioTranslationsService) {
+    var activeContentId = null;
+    return {
+      getActiveContentId: function() {
+        return activeContentId;
+      },
+      setActiveContentId: function(contentId) {
+        var allContentIds = StateContentIdsToAudioTranslationsService.displayed
+          .getAllContentId();
+        if (allContentIds.indexOf(contentId) < 0) {
```

Comment:
`=== -1` is better

Or maybe we can start using .includes()? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes Seems like browser compatibility is good.

Reply:
@seanlip @DubeySandeep Please note that .includes() is not supported in Internet Explorer.
Thanks.

Reply:
@rakshitkumarcse, Thanks for looking into it. 

@seanlip, I've added `=== -1`, let me know if you want to have `includes()`?

Reply:
-1 is good. Thanks @rakshitkumarcse for catching that! My first inclination was that we don't support IE, but I looked up the browser market share and it looks like it accounts for ~10% of usage so probably better to be cautious.